### ISSUE TITLE
Is it still good to go?

### ISSUE THREAD

**Issue by ghost** [Notos/seedbox-from-scratch#18]
It's not an issue but I just wanted to know whether it's still good to go? I am going to try this on a fresh dedi and this is my first dedicated experience.

(The latest file - one 6 month old and another 1 year old; rest are 2 years old)

**Comment by dannyti** [comment]
It is still good except for autodl irssi and latest versions. 
I have modified it for correct autodl and latest OS 
https://github.com/dannyti/sboxsetup/blob/master/sbfrmsc-dti.sh
Try it.

**Comment by derooy** [comment]
If you're wondering why the files are so old.  The latest linux distro that is supported is ubuntu 12.04 which was released about 2 years ago.  As long as you use the ubuntu that is supported you shouldn't have an issue (=

edit: it's actually 12.10; but you get the idea :p
edit: I feel mega awkward the way that I answered this.  I assumed you were new, didn't check your account :(  No offence intended
